raiden/app.py

```diff
@@ -1,97 +0,0 @@
-
-log = structlog.get_logger(__name__)
-
-
-class App:  # pylint: disable=too-few-public-methods
-    def __init__(
-        self,
-        config: RaidenConfig,
-        rpc_client: JSONRPCClient,
-        proxy_manager: ProxyManager,
-        query_start_block: BlockNumber,
-        default_registry: TokenNetworkRegistry,
-        default_secret_registry: SecretRegistry,
-        default_service_registry: Optional[ServiceRegistry],
-        default_user_deposit: Optional[UserDeposit],
-        default_one_to_n_address: Optional[OneToNAddress],
-        default_msc_address: Optional[MonitoringServiceAddress],
-        transport: MatrixTransport,
-        raiden_event_handler: EventHandler,
-        message_handler: MessageHandler,
-        routing_mode: RoutingMode,
-        api_server: APIServer = None,
-    ):
-        raiden = RaidenService(
-            rpc_client=rpc_client,
-            proxy_manager=proxy_manager,
-            query_start_block=query_start_block,
-            default_registry=default_registry,
-            default_secret_registry=default_secret_registry,
-            default_service_registry=default_service_registry,
-            default_user_deposit=default_user_deposit,
-            default_one_to_n_address=default_one_to_n_address,
-            default_msc_address=default_msc_address,
-            transport=transport,
-            raiden_event_handler=raiden_event_handler,
-            message_handler=message_handler,
-            routing_mode=routing_mode,
-            config=config,
-            api_server=api_server,
-        )
-
-        # check that the settlement timeout fits the limits of the contract
-        settlement_timeout_min = default_registry.settlement_timeout_min(BLOCK_ID_LATEST)
-        settlement_timeout_max = default_registry.settlement_timeout_max(BLOCK_ID_LATEST)
-        invalid_settle_timeout = (
-            config.settle_timeout < settlement_timeout_min
-            or config.settle_timeout > settlement_timeout_max
-            or config.settle_timeout < config.reveal_timeout * 2
-        )
-        if invalid_settle_timeout:
-            raise InvalidSettleTimeout(
-                (
-                    "Settlement timeout for Registry contract {} must "
-                    "be in range [{}, {}], is {}"
-                ).format(
-                    to_checksum_address(default_registry.address),
-                    settlement_timeout_min,
-                    settlement_timeout_max,
-                    config.settle_timeout,
-                )
-            )
-
-        self.config = config
-        self.raiden = raiden
-
-    def __repr__(self) -> str:
-        return "<{} {}>".format(self.__class__.__name__, to_checksum_address(self.raiden.address))
-
-    def start(self) -> None:
-        """ Start the raiden app. """
-        if self.raiden.stop_event.is_set():
-            self.raiden.start()
-            log.info("Raiden started", node=to_checksum_address(self.raiden.address))
```

Comment:
Should we keep those log lines?

Reply:
There are equivalent log lines for the service: https://github.com/raiden-network/raiden/blob/develop/raiden/raiden_service.py#L447